CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # OSIAM addon-administration
 
+## Unreleased
+
+### Fixes
+
+- Fix handling of editing groups and keep the members.
```

Comment:
This doesn't make it clear to me what is fixed. Why not use the commit message or write it like "Fix: [original issue title]".